### PR TITLE
Run check-multi-machine-key on atomic

### PR DESCRIPTION
**Pull request by petervo**

### CODE REVIEW

File: src/bridge/bridge.c [hunk 1]
```diff
@@ -391,17 +394,39 @@ start_ssh_agent (void)
...
 
-  flags = G_SPAWN_SEARCH_PATH | G_SPAWN_STDOUT_TO_DEV_NULL;
-  g_spawn_async (NULL, agent_argv, NULL, flags,
-                 setup_ssh_agent, GINT_TO_POINTER (-1), &pid, &error);
-
-  if (error != NULL)
+  if (!g_spawn_sync (NULL, agent_argv, NULL,
+                     G_SPAWN_SEARCH_PATH, setup_ssh_agent,
+                     GINT_TO_POINTER (-1),
+                     &agent_output, &agent_error,
```
Comment (stefwalter) [inline src/bridge/bridge.c]: Shouldn't we just let stderr fall through to output?

File: src/bridge/bridge.c [hunk 2]
```diff
@@ -391,17 +394,39 @@ start_ssh_agent (void)
...
+    }
+
+  if (!g_spawn_check_exit_status (status, &error))
+    {
+      g_warning ("couldn't start %s: %s: %s", agent_argv[0],
+                 error->message, agent_error);
+      goto out;
+    }
+
+  pid_line = strstr(agent_output, "SSH_AGENT_PID=");
```
Comment (stefwalter) [inline src/bridge/bridge.c]: Needs a space after function name.

File: src/bridge/bridge.c (hunk 1 above)
Comment (petervo) [inline src/bridge/bridge.c]: I don't think g_spawn_sync supports that, you either supply a pointer or use G_SPAWN_STDERR_TO_DEV_NULL

Comment (stefwalter) [inline src/bridge/bridge.c]: That's nasty.

Comment (stefwalter) [inline src/bridge/bridge.c]: That's horrible.

**Pull request closed without merging**